docs/index.md

```diff
@@ -56,6 +56,10 @@ EOF
 <!-- schema generated by tfplugindocs -->
 ## Schema
 
+### Required
```

Comment:
I am confused as to why someone had documented this as option, considering the variable looks like it is required.